backend/btrixcloud/operator.py

```diff
@@ -512,6 +515,11 @@ async def mark_finished(
         status.state = state
         status.finished = to_k8s_date(finished)
 
+        if state in FAILED_STATES:
```

Comment:
Hm, i wonder if instead of adding collections on crawl creation, and then removing if crawl fails, we add them when the crawl is completed?

Reply:
Yeah in the fresh light of day I agree, that'd be better!